### DISCUSSION TITLE
Incorrect package version

### DISCUSSION THREAD

**Discussion by wendellestradairely** [vmware/clarity#6227] [Q&A] [answered]
When you `npm i @clr/<package>@latest`, the version number becomes 12.0
![image](https://user-images.githubusercontent.com/50942719/128456211-9cd64935-4365-4072-96cd-4f959d360f23.png)

**Comment by bbogdanov** [comment, marked as answer]
Hi @wendellestradairely, 

That's correct! 12.0.1 is the latest version of `@clr/*` packages.

https://www.npmjs.com/package/@clr/angular

**Reply by wendellestradairely** [reply]
@bbogdanov Okay, got it. Thanks. But the documentation shows 5.5.1
![image](https://user-images.githubusercontent.com/50942719/128473844-3a93231a-20b3-472f-9e49-3b481a99d874.png)

**Reply by bbogdanov** [reply]
We will publish a blog post soon about what we are doing with the website but long story short we have two websites now.

https://clarity.design - That represents Clarity Core and related information about Clarity Design System. 

https://angular.clarity.design - That represents Clarity Angular <- This is the one you should be looking for related to Clarity Angular updates
